### DISCUSSION TITLE
[11 Settembre 2020] - Modifica CPU

### DISCUSSION THREAD

**Discussion by rimaout** [CS-Swap/Architettura-degli-Elaboratori#29] [Modifica Architettura]
<img width="1171" alt="image" src="https://github.com/CS-Swap/Architettura-degli-Elaboratori/assets/116072651/3e1ad868-eacc-4e3d-8811-4cbf8a230774">

<img width="1062" alt="image" src="https://github.com/CS-Swap/Architettura-degli-Elaboratori/assets/116072651/2917f936-2dff-43ec-a383-c7d9cb7d722f">

<details><summary>Soluzione Prof</summary><p> 
<img width="936" alt="image" src="https://github.com/CS-Swap/Architettura-degli-Elaboratori/assets/116072651/7a0bc463-22e0-4975-b9af-cca48a7c4d95">
<img width="1032" alt="image" src="https://github.com/CS-Swap/Architettura-degli-Elaboratori/assets/116072651/345afbac-eaed-4782-a7a5-694ba7867705">
<img width="611" alt="image" src="https://github.com/CS-Swap/Architettura-degli-Elaboratori/assets/116072651/7c8c021c-3154-40e0-998f-1ad98f605557">

</p></details>

**Comment by notedo** [comment]
![ESERCIZI - Architettura-57](https://github.com/CS-Swap/Architettura-degli-Elaboratori/assets/110790224/d8cc0f1d-e1a2-46f1-a087-033f46654eae)
![IMG_0494](https://github.com/CS-Swap/Architettura-degli-Elaboratori/assets/110790224/24852631-2ee9-4fa4-890e-412938e436a7)
